llpc/test/shaderdb/fuzzer/GraphicsFuzz_TestOpCopyObject2.spvasm

```diff
@@ -0,0 +1,45 @@
+; BEGIN_SHADERTEST
```

Comment:
What is the difference from TestOpCopyObject? I think "2" is somewhat unclear. Maybe use more descriptive words